tests/sentinel/tests/00-base.tcl

```diff
@@ -2,6 +2,10 @@
 source "../tests/includes/start-init-tests.tcl"
 source "../tests/includes/init-tests.tcl"
 
+foreach_sentinel_id id {
```

Comment:
The name of this function suggests that it performs some kind of loop, but I can't see that in the body.  In fact, I cannot see any relationship between the function name and what it does.  Perhaps a name like set-default-down-after-to-1000 would be better.